src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
 body {
   margin: 0;
 }
+
+.card {
+  max-width: 200px;
+  height: 406px;
+
+  box-sizing: border-box;
+
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  position: relative;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card__picture {
+  position: absolute;
+  top: 33px;
+
+  max-width: 160px;
+
+}
+
+.card__item-model {
+  position: absolute;
+  top: 207px;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  width: 85%;
```

Comment:
Apply padding to the container so you don't need to use odd values. This way, the card can span 100%, and the padding will take care of the rest.